capi/geos_c.h.in

```diff
@@ -631,6 +631,9 @@ extern GEOSGeometry GEOS_DLL *GEOSPolygonize_full_r(GEOSContextHandle_t handle,
                               const GEOSGeometry* input, GEOSGeometry** cuts,
                               GEOSGeometry** dangles, GEOSGeometry** invalidRings);
 
+extern GEOSGeometry GEOS_DLL *GEOSBuildArea_r(GEOSContextHandle_t handle,
```

Comment:
I don't think the name "BuildArea" does a good job differentiating itself from "Polygonize" (I realize the name comes from PostGIS). Would something like `GEOSPolygonizeExcludeHoles` be more clear (and avoid changing the signature to `GEOSPolygonize`)?

Reply:
Taken into account